### PR TITLE
Added basic navigation information

### PR DESCRIPTION
**Pull request by MisterIXI**

### CODE REVIEW

File: Navigation.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## launching navigation with a map file
+```bash
+roslaunch turtlebot3_navigation turtlebot3_navigation.launch map_file:=/home/yannik/map_new.yaml```
```
Comment (v0rl0ck) [inline Navigation.md]: Move the ``` to the next line, formatting is broken

File: Navigation.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+## finding the ros locations
+```bash
+rospack find turtlebot3_navigation```
```
Comment (v0rl0ck) [inline Navigation.md]: Move the ``` to the next line, formatting is broken

File: Navigation.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## changing speed values in the folders
+/opt/ros/melodic/share/turtlebot3_navigation/param
```
Comment (v0rl0ck) [inline Navigation.md]: Add quotes to mark the path in the markdown

**Review by v0rl0ck — CHANGES_REQUESTED** [review]

File: Navigation.md (hunk 2 above)
Comment (MisterIXI) [reply Navigation.md]: Done

File: Navigation.md (hunk 3 above)
Comment (MisterIXI) [reply Navigation.md]: Done

File: Navigation.md (hunk 1 above)
Comment (MisterIXI) [reply Navigation.md]: Done

**Review by v0rl0ck — APPROVED** [review]
Looks good now :D

**Pull request merged**